src/shared/orgListUtil.ts

```diff
@@ -52,9 +52,7 @@ export class OrgListUtil {
     const [nonScratchOrgs, scratchOrgs] = await Promise.all([
       Promise.all(
         orgs.nonScratchOrgs.map(async (fields) => {
-          if (flags.skipconnectionstatus) {
-            fields.connectedStatus = fields.connectedStatus ?? 'Unknown';
-          } else {
+          if (!flags.skipconnectionstatus) {
```

Comment:
@mshanemc is there a test to cover this?

Reply:
https://github.com/salesforcecli/plugin-org/blob/c53fba1f9acd6ceb5318bf73727ae851162fee54/test/shared/orgListUtil.test.ts#L112

the command is just passing that flag to orgListUtil so the test is at that level instead of on the command.

Reply:
Another way to put it, is there a way this bug could have been caught with a test?

Reply:
that test now asserts that `--skipconnectionstatus` produces nonScratchOrgs with no `connectedStatus` in the json

Reply:
@mshanemc Does the test below represent verification of the correct behavior when `flags.skipconnectionstatus` is false?

https://github.com/salesforcecli/plugin-org/blob/de91ff54a7db5d78460293e8889cbf9b703f8514/test/shared/orgListUtil.test.ts#L102

Reply:
when migrating this command, I tried to keep the tests on orgListUtil as close to the original as possible.

I don't like the original either, so I'm gonna fix them.

Reply:
better?